runtime/build.rs

```diff
@@ -0,0 +1,7 @@
+use std::env;
+
+fn main() {
+    if env::var("CARGO_FEATURE_NVTX").is_ok() {
```

Comment:
Let’s try moving this to runtime, and remove the cargo feature.

At runtime, measure.rs can test for an nvtx env var and use a dlopen wrapper like https://docs.rs/dlopen/0.1.7/dlopen/symbor/index.html to load 

Also let’s move measure into its own crate so it can be used anywhere in the repo (this also gets easier if we don’t use a cargo feature)

Reply:
good ideas, let me try that.

Reply:
I'll add the nvtx-less one first, then have a new PR which adds it.